roqoqo-qasm/Cargo.toml

```diff
@@ -21,6 +21,9 @@ doctest = false
 roqoqo = { version = "1.4", features = ["serialize"] }
 qoqo_calculator = { version = "1.1" }
 ndarray = "0.15"
+pest = "2.5"
+pest_derive = "2.5"
+num-complex = "0.4"
```

Comment:
for now we could put the qasm import behind a `unstable_qasm_import` feature gate, make these optional and only include the import related files if the feature is activated.